.travis.yml

```diff
@@ -9,6 +9,9 @@ services:
 env:
   - TOX_ENV=py27
 
+git:
+  submodules: false
+
```

Comment:
As per Travis docs: https://docs.travis-ci.com/user/customizing-the-build/#sts=Git Submodules